### PR TITLE
Fix two bugs: printing vectors of size 0 + processing single byte lines

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

**Comment by lluiscampos** [comment]
@kacf I think these two bugs are in production code and we shall release the fixes.

**Comment by lluiscampos** [comment]
@mender-test-bot start pipeline

**Review by vpodzime — APPROVED** [review]
~betting~ *being* in the second commit message. ACK otherwise.

**Review by kacf — APPROVED** [review]

File: src/common/common.hpp [hunk 1]
```diff
@@ -107,10 +107,13 @@ static inline bool VectorContainsString(const vector<string> &vec, const string
 static inline string StringVectorToString(const vector<string> &vec, const string delim = ",") {
 	string ret = "{";
 	auto sz = vec.size();
-	for (decltype(sz) i = 0; i < (sz - 1); i++) {
-		ret += "\"" + vec[i] + "\"" + delim;
+	if (sz > 0) {
+		for (decltype(sz) i = 0; i < (sz - 1); i++) {
```
[the remark this answers is not in the document]
Comment (vpodzime) [reply src/common/common.hpp:111]: C++ was safe, with great bounds checking....they said... :wink:

**Comment by lluiscampos** [comment]
The last force-push only changed commit message formalities - the [previous pipeline](https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/1213165501) is valid.

**Pull request merged**